changelog.txt

```diff
@@ -1,5 +1,8 @@
 *** WooCommerce Payments Changelog ***
 
+= 1.5.0 - 2020-xx-xx =
+* Add - Display payment method details on account subscriptions pages.
```

Comment:
Should we also update the `readme.txt`? Or is it updated as part of the release?

Reply:
Good point, TBH I thought we agreed on updating changelog only but saw in some PRs readme is updated as well. So I've added the entry to readme in https://github.com/Automattic/woocommerce-payments/pull/893/commits/b64125c8bb2f36205448f18870146f90423b9201. Also decided to be a bit more flexible with next version numbers :)